src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSUtils.java

```diff
@@ -57,24 +51,6 @@ public static void logItemizedWarning(final Logger logger, final Collection<Stri
     }
 
     /**
-     * Writes two objects using Kryo to specified local file path.
```

Comment:
This got moved to PSUtils.java. Using String.join() instead since it's fewer lines.